src/Service/PaymentsService.php

```diff
@@ -54,6 +55,20 @@ public function createRecurrentPayment(RecurrentPayment $payment)
 	}
 
 	/**
+	 * @param string $recurrencePaymentId
+	 * @param RecurringPayment $payment
+	 * @return Response
+	 */
+	public function createRecurringPayment($recurrencePaymentId, RecurringPayment $payment)
```

Comment:
Yep, but it makes a different API call - same as CreatePayment method, but with "recurrence" data sets - and serves to authorize subsequent recurring payments.

This method is used to initiate a recurring payment based on that authorization one. It needs reference to authorization payment (by it's id) as input parameter